src/core/core.scale.js

```diff
@@ -158,17 +158,12 @@ function getTickMarkLength(options) {
 	return options.drawTicks ? options.tickMarkLength : 0;
 }
 
-function getScaleLabelHeight(options) {
-	var font, padding;
-
-	if (!options.display) {
+function getScaleLabelHeight(labelOpts) {
+	if (!labelOpts.display) {
```

Comment:
I'd alias `parseFont` and `toPadding` (both user more than once) and remove one more return:

```js
return !labelOpts.display ? 0 : parseFont(labelOpts).lineHeight + toPadding(labelOpts.padding).height;
```

Reply:
done